cmd/sriov/main.go

```diff
@@ -209,6 +193,32 @@ func cmdAdd(args *skel.CmdArgs) error {
 		return fmt.Errorf("error saving the pci allocation for vf pci address %s: %v", netConf.DeviceID, err)
 	}
 
+	if doAnnounce {
```

Comment:
why do we need to move this  logic here ? just to "give it more time" ?

Reply:
To not do the non-criticial, potentially long blocking action in between.

The commit message attempts to and should explain that. Is that somehow unclear.

Reply:
it is clear.